### PR TITLE
Debug Logs

### PR DESCRIPTION
**Pull request by damoodamoo**
WIP

### CODE REVIEW

File: .github/workflows/deploy_tre.yml [hunk 1]
```diff
@@ -146,3 +146,4 @@ jobs:
       TRE_ADDRESS_SPACE: ${{ secrets.TRE_ADDRESS_SPACE }}
       TRE_ID: ${{ format('tre{0}', needs.prepare-not-main.outputs.refid) }}
       CI_CACHE_ACR_NAME: ${{ secrets.ACR_NAME }}
+      TF_LOG: ${{ secrets.TF_LOG }}
```
Comment (ross-p-smith) [inline .github/workflows/deploy_tre.yml:149]: is the value a secret? Would it be better as a normal input and then you could see its value in the logs? secrets end up being ***

Comment (stuartleeks) [reply .github/workflows/deploy_tre.yml:149]: If it's coming from a GH secret in the repo, doesn't that get masked regardless of how you pass it in to other actions etc?

**Pull request closed without merging**